program/rust/src/rust_oracle.rs

```diff
@@ -170,6 +175,57 @@ pub fn add_price(
     Ok(SUCCESS)
 }
 
+/// add a publisher to a price account
+/// accounts[0] funding account                                   [signer writable]
+/// accounts[1] price account to add the publisher to             [signer writable]
+pub fn add_publisher(
+    program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    instruction_data: &[u8],
+) -> OracleResult {
+    let cmd_args = load::<cmd_add_publisher_t>(instruction_data)?;
+
+    if instruction_data.len() != size_of::<cmd_add_publisher_t>() || pubkey_is_zero(&cmd_args.pub_)
+    {
+        return Err(ProgramError::InvalidArgument);
+    }
+
+    let [_funding_account, price_account] = match accounts {
```

Comment:
_ prefix indicates the variable isn't used later in the function, but _funding_account is